### ISSUE TITLE
Cannot update contact person (to any not manager user) via API

### ISSUE THREAD

**Issue by jiho** [ecotaxa/ecotaxa_dev#597]
Pseudo code/steps
```
GET projects/10
Parse JSON and get contact from project 10
GET projects/16
Set empty contact in project 16 to that of project 10
PUT projects/16
GET projects/16
```

Expected: Contact in 16 is the same as in 10
Obtained: Contact in 16 is still NULL.

The same approach works with comments or other simple fields.

**Comment by grololo06** [comment]
The contact person is checked against the list of managers for the project, so if one sets a contact who **is not a manager** on the project then the contact is simply ignored (bad).
Options:
- automatically create/set as as manager the contact if provided
- throw an error in such case
Let me know.

**Comment by picheral** [comment]
I vote for 1

**Comment by jiho** [comment]
I vote for the **error** actually there. It would be the same as if someone was trying to set a non-manager as contact via the UI (which is not possible because the radio button is deactivated). In any case, it is always better to fail explicitly rather than silently.

And no, actually, in project 10 the contact is me and I was trying to set the contact of project 16 to me too. I am manager of project 16. It seems the fix for #596 fixed this too. I could update contact and license on 90 projects through the API. Yay!

Closing this and opening another issue, more specific.
